ciw/simulation.py

```diff
@@ -86,6 +86,31 @@ def choose_tracker(self, tracker, deadlock_detector):
             return NaiveTracker(self)
         return StateTracker(self)
 
+    def detect_deadlock(self):
```

Comment:
Yeah this shouldn't be there (It's not in a deadlock_detector object)
This also the reason coverage went down. Can you remove it please?